src/main/java/org/mifos/pheeBillPay/api/implementation/BillInquiryController.java

```diff
@@ -26,6 +26,11 @@ public ResponseEntity<BillInquiryResponseDTO> billInquiry(String tenantId, Strin
         } catch (Exception e) {
             return ResponseEntity.badRequest().build();
         }
-        return ResponseEntity.status(HttpStatus.ACCEPTED).body(billInquiryResponseDTO);
+        if(billInquiryResponseDTO.getTransactionId().equals("NA") ||
```

Comment:
use Enums for better readability. 
why does the transaction Id been passed as NA